index.html

```diff
@@ -482,6 +482,60 @@ <h3 id="examples">More examples</h3>
   }
 }])</code></pre>
     </li>
+
+
+    <li class="timer-functions" id="timer-functions">
+      <div class="ui">
+        <p>Timer functions example</p>
+        <button aria-label="Try me! Example: Timer functions">Try me!</button>
+      </div>
+      <pre class="code-sample">
+<code>  let timerInterval
+  Swal({
+    title: 'Auto close alert!',
+    html: 'I will close in &lt;strong&gt;&lt;/strong&gt; seconds.' + '&lt;br/&gt;&lt;br/&gt;' + 
+    '&lt;button id="increase-timer"&gt;I need 5 more seconds!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="stop-timer"&gt;Please stop the timer!!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="resume-timer" disabled&gt;Phew... you can restart now!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="toggle-timer"&gt;Toggle&lt;/button&gt;',
+    timer: 10000,
+    backdrop: false,
+    onOpen: () => {
+      Swal.showLoading()
+
+      function toggleButtons () {
+        Swal.getContent().querySelector('#stop-timer').disabled = !Swal.getContent().querySelector('#stop-timer').disabled
```

Comment:
`Swal.getContent().querySelector('#stop-timer')` is repeated multiple times, that violates the DRY principle, consider creating a variable:

```js
const content = Swal.getContent()
const $ = content.querySelector.bind(content)

const stop = $('#stop-timer')
const resume = $('#resume-timer')
const toggle = $('#toggle-timer')
const increase = $('#increase-timer')
```

Reply:
Done in a90f6d6